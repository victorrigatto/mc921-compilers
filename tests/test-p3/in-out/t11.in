int main() {
    int y = {1, 2};
    return 0;
}
