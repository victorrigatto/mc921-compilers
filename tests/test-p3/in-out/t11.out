SemanticError: y initialization must be a single element @ 2:9
